modules/systemtests/src-test/org/ejbca/core/protocol/ocsp/ProtocolOcspHttpSystemTest.java

```diff
@@ -505,6 +505,7 @@ public void test08OcspEcdsaGood() throws Exception {
         } finally {
             endEntityManagementSession.deleteUser(admin, "ocsptest");
             CryptoTokenTestUtils.removeCryptoToken(admin, caInfo.getCAToken().getCryptoTokenId());
+            internalCertStoreSession.removeCertificatesByIssuer( caInfo.getSubjectDN());
```

Comment:
removing the crypto token and certsByIssuer is actually done in tearDown as far as I can see?
a "removeCertificatesByUsername" after the deleteUser could look good, but is also not needed as CaTestUtils removes all issued certificates now.